lnst/Recipes/ENRT/ConfigMixins/CommonHWSubConfigMixin.py

```diff
@@ -25,4 +25,3 @@ class CommonHWSubConfigMixin(
     convenience. For more details, see the documentation of the individual
     ancestor classes.
     """
-    pass
```

Comment:
Just making a note in case we decide to keep this as in https://github.com/LNST-project/lnst/pull/244/commits/164b63a9f18b7557e489dffe815676c9ba0de9f6#r886821818